packages/flutter_tools/test/general.shard/project_test.dart

```diff
@@ -776,6 +776,7 @@ void testInMemory(String description, Future<void> testMethod()) {
   final FileSystem testFileSystem = MemoryFileSystem(
     style: globals.platform.isWindows ? FileSystemStyle.windows : FileSystemStyle.posix,
   );
+  testFileSystem.file('.packages').writeAsStringSync('\n');
```

Comment:
🕵️‍♂️ 

This was throwing in the packageConfig loader in the loadPlugins way down below